Snippets/Rabbit/Rabbit_4/Usage.cs

```diff
@@ -151,4 +151,25 @@ void SetClientCertificates(EndpointConfiguration endpointConfiguration)
         #endregion
     }
 
-}
\ No newline at end of file
+    void DisableTimeoutManager(EndpointConfiguration endpointConfiguration)
+    {
+        #region rabbitmq-delay-disable-timeout-manager 4.3
```

Comment:
I was trying to scope the name of the snippet, which some of the others already do as well.